external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaRelation.scala

```diff
@@ -143,4 +143,6 @@ private[kafka010] class KafkaRelation(
         validateTopicPartitions(partitions, partitionOffsets)
     }
   }
+
+  override def toString: String = "kafka"
 }
```

Comment:
Hehe...been thinking about it too, but thought it'd be too much for such a simple change.

It merits a JIRA issue though where one had to find out what and how to display. After all, you could read from multiple topics with different Kafka brokers and offsets. I was worried it'd be too much to handle as part of this issue.

I'm up for working on the other issue, but don't think we should hold this one up. WDYT?